lib/translit.rb

```diff
@@ -65,21 +75,21 @@ def self.slavic_language(text)
     end
   end
 
-  # Unsupported latin: "ä"=>["Э","э"],
+  # Unsupported latin: "ä"=>["Э","э"], "ü"=>["Ю","ю"],
   def self.english_to_ukrainian
-    { "a"=>["А","а"], "b"=>["Б","б"], "v"=>["В","в"], "h"=>["Г","г"], "g"=>["Ґ","ґ"], "d"=>["Д","д"], "e"=>["Е","е"], "ye"=>["Є","є"], "je"=>["Є","є"], "zh"=>["Ж","ж"],
+    @english_to_ukrainian ||= latin_cases({ "a"=>["А","а"], "b"=>["Б","б"], "v"=>["В","в"], "h"=>["Г","г"], "g"=>["Ґ","ґ"], "d"=>["Д","д"], "e"=>["Е","е"], "ye"=>["Є","є"], "je"=>["Є","є"], "zh"=>["Ж","ж"],
       "z"=>["З","з"], "i"=>["І","і"], "yi"=>["Ї","ї"], "j"=>["Й","й"], "k"=>["К","к"], "l"=>["Л","л"], "m"=>["М","м"], "n"=>["Н","н"], "o"=>["О","о"], "p"=>["П","п"], "r"=>["Р","р"],
       "s"=>["С","с"], "t"=>["Т","т"], "u"=>["У","у"], "f"=>["Ф","ф"], "kh"=>["Х","х"], "x"=>["Кс","кс"], "ts"=>["Ц","ц"], "ch"=>["Ч","ч"], "sh"=>["Ш","ш"], "w"=>["В","в"],
       "shch"=>["Щ","щ"], "sch"=>["Щ","щ"], "y"=>["И","и"], "'"=>["Ь","ь"], "yu"=>["Ю","ю"], "ju"=>["Ю","ю"],
-      "ü"=>["Ю","ю"], "ya"=>["Я","я"], "ja"=>["Я","я"], "q"=>["К","к"]}
+      "ü"=>["Ю","ю"], "ya"=>["Я","я"], "ja"=>["Я","я"], "q"=>["К","к"]})
   end
 
   def self.english_to_russian
-    { "a"=>["А","а"], "b"=>["Б","б"], "v"=>["В","в"], "g"=>["Г","г"], "d"=>["Д","д"], "e"=>["Е","е"], "yo"=>["Ё","ё"], "jo"=>["Ё","ё"], "ö"=>["Ё","ё"], "zh"=>["Ж","ж"],
+    @english_to_russian ||= latin_cases({ "a"=>["А","а"], "b"=>["Б","б"], "v"=>["В","в"], "g"=>["Г","г"], "d"=>["Д","д"], "e"=>["Е","е"], "yo"=>["Ё","ё"], "jo"=>["Ё","ё"], "ö"=>["Ё","ё"], "zh"=>["Ж","ж"],
```

Comment:
Make sure both hashes are passed to the latin_cases method so both cases are listed as keys in the resulting hash.